docs/examples/pipeline/query_pipeline_async.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        retrievers[str(chunk_size)] = vector_index.as_retriever()
</pre><p>I don't think the retrievers dict is used. Can we remove it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9967/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>